frontend/src/utilities/notebookControllerUtils.ts

```diff
@@ -42,11 +42,12 @@ export const generateEnvVarFileNameFromUsername = (username: string): string =>
  */
 export const verifyResource = async <T>(
   name: string,
-  fetchFunc: (resourceName: string) => Promise<T>,
+  namespace: string,
+  fetchFunc: (projectName: string, resourceName: string) => Promise<T>,
```

Comment:
hah, we are reworking these types too much -- we should really look to make a single type and reuse it everywhere. The Spawner page had similar changes -- usually that's an indication we are failing DRY